hy/models/complex.py

```diff
@@ -30,3 +30,5 @@ class HyComplex(HyObject, complex):
     def __new__(cls, number, *args, **kwargs):
         number = complex(number)
         return super(HyComplex, cls).__new__(cls, number)
+
+_wrappers[complex] = HyComplex
```

Comment:
Yeah, I'm OK leaving them in the modules. @berkerpeksag any other thoughts on this?